tests/import_test.go

```diff
@@ -129,6 +129,58 @@ var _ = Describe("[rfe_id:1115][crit:high][vendor:cnv-qe@redhat.com][level:compo
 	})
 })
 
+var _ = Describe("Importer respects node placement", func() {
+	var cr *cdiv1.CDI
+	var oldSpec *cdiv1.CDISpec
+	f := framework.NewFramework(namespacePrefix)
+
+	// An image that fails import
+	invalidQcowLargeSize := func() string {
+		return fmt.Sprintf(utils.InvalidQcowImagesURL+"invalid-qcow-large-size.img", f.CdiInstallNs)
+	}
+
+	BeforeEach(func() {
+		var err error
+		cr, err = f.CdiClient.CdiV1beta1().CDIs().Get(context.TODO(), "cdi", metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+
+		oldSpec = cr.Spec.DeepCopy()
+		nodes, err := f.K8sClient.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+		Expect(nodes.Items).ToNot(BeEmpty(), "There should be some compute node")
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	AfterEach(func() {
+		cr, err := f.CdiClient.CdiV1beta1().CDIs().Get(context.TODO(), "cdi", metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+
+		cr.Spec = *oldSpec.DeepCopy()
+		_, err = f.CdiClient.CdiV1beta1().CDIs().Update(context.TODO(), cr, metav1.UpdateOptions{})
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	It("Should create import pod with node placement", func() {
+		cr.Spec.Workloads = tests.TestNodePlacementValues(f)
+		_, err := f.CdiClient.CdiV1beta1().CDIs().Update(context.TODO(), cr, metav1.UpdateOptions{})
+
+		dv := utils.NewDataVolumeWithHTTPImport("node-placement-test", "100Mi", invalidQcowLargeSize())
```

Comment:
Verify the CR has the node placement part applied before creating the new data volume, right now we have a race where we could potentially create the pod spec before the update to the the CDI object has been recorded.